packages/core/src/utils/StepActionArgs.ts

```diff
@@ -0,0 +1,40 @@
+import { BaseLocator } from './../page/Locator'
+import {
+	ElementVisibilityCondition,
+	ElementTextCondition,
+	ElementTextNotMatchCondition,
+	ElementStateCondition,
+	ElementSelectedCondition,
+} from '../page/conditions/'
+
+export const StepActionArgs = (args: any[]): string => {
+	let result = ''
+
+	const isInstanceOfCondition = (arg: any): boolean => {
+		return (
+			arg instanceof ElementVisibilityCondition ||
+			arg instanceof ElementTextCondition ||
+			arg instanceof ElementTextNotMatchCondition ||
+			arg instanceof ElementStateCondition ||
+			arg instanceof ElementSelectedCondition
+		)
+	}
+
+	for (const arg of args) {
+		if (typeof arg === 'string') {
+			result = `'${arg}'`
+			break
+		}
+
+		if (isInstanceOfCondition(arg)) {
+			const locator: string = JSON.parse(JSON.stringify(arg.locator)).errorString
+			result = `Util.${arg.desc}(${locator})`
```

Comment:
```suggestion
			result = `Until.${arg.desc}(${locator})`
```
this might be an issue if we hard `Until` here

Reply:
Sorry for my wording mistake.